src/Pages/ProductDetailPage.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import styled from "styled-components";
+import { useParams } from "react-router";
+
+import { useFeaturedBanners } from "utils/hooks/useFeaturedBanners";
+import { BannerContainer } from "Components/HomePage/styled";
+import Slider from "Components/Slider/Slider";
+import Button from "Components/Button";
+
+const ProductDetailPageContainer = styled.section`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`;
+
+const DetailsContainer = styled.article`
+    display: flex;
+    flex-direction: column;
+    max-width: 50vw;
+`;
+
+const ProductDetailPage = () => {
+
+    let {id} = useParams();
+    let productDetails;
+
+    const result = useFeaturedBanners(null, null, "productId", id );
+
+    if(!result.isLoading){
+        productDetails = result.data.results[0];
+        const {
+            data:{
+                description,
+                images,
+                name,
+                price,
+                sku,
+                specs,
+                stock,
+                category
+            },
```

Comment:
Remember to avoid deep destructuring like this. It is easier to read if you do a second destructuring for `data`. Or even better, if you do:

```javascript
const { description, ... } = productDetails.data;
```